django/contrib/admin/sites.py

```diff
@@ -63,6 +64,21 @@ def __init__(self, name='admin'):
         self.name = name
         self._actions = {'delete_selected': actions.delete_selected}
         self._global_actions = self._actions.copy()
+        all_sites.add(self)
+
+    def check(self):
+        """
+        Run the system checks of the underlying admin objects.
+        """
+        if not settings.DEBUG:
+            return []
+
+        errors = []
+        admin_objects_to_check = [o for o in self._registry.values() if o.__class__ is not ModelAdmin]
```

Comment:
@timgraham I don't think so. See: https://github.com/django/django/blob/master/django/contrib/admin/sites.py#L105 If there are options associated. A new admin subclass of `admin_class`  is dynamically created if there are options defined in the `register` call, thus ensuring the checks are executed for this subclass.